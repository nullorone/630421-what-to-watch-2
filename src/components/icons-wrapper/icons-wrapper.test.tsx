import * as React from "react";
import renderer from "react-test-renderer";
import IconsWrapper from "./icons-wrapper";

describe(`Make snapshot`, () => {
  const initProps = {
    iconNames: [`ADD`, `PAUSE`]
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <IconsWrapper {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
