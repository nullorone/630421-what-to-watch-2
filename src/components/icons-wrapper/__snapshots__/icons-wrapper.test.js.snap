// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<div
  className="visually-hidden"
>
  <svg
    xmlns="http://www.w3.org/2000/svg"
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <symbol
      id="add"
      viewBox="0 0 19 20"
    >
      <title>
        +
      </title>
      <desc>
        Created with Sketch.
      </desc>
      <g
        fill="none"
        fillRule="evenodd"
        id="Page-1"
        stroke="none"
        strokeWidth="1"
      >
        <polygon
          fill="#EEE5B5"
          id="+"
          points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
        />
      </g>
    </symbol>
    <symbol
      id="pause"
      viewBox="0 0 14 21"
    >
      <g
        fill="none"
        fillRule="evenodd"
        id="Artboard"
        stroke="none"
        strokeWidth="1"
      >
        <polygon
          fill="#EEE5B5"
          fillRule="nonzero"
          id="Line"
          points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
        />
        <polygon
          fill="#EEE5B5"
          fillRule="nonzero"
          id="Line"
          points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
        />
      </g>
    </symbol>
  </svg>
</div>
`;
